packages/flutter/lib/src/material/bottom_app_bar.dart

```diff
@@ -187,15 +189,16 @@ class _BottomAppBarState extends State<BottomAppBar> {
       height: height,
       child: PhysicalShape(
         clipper: clipper,
-        elevation: elevation,
         color: effectiveColor,
         clipBehavior: widget.clipBehavior,
+        elevation: isMaterial3 ? 0 : elevation,
```

Comment:
Ah actually, this also prevents higher elevation, which we still want to allow

Reply:
It can be elevated (for tinting effects) via the Material elevation property (L198). 

Or do you mean we should preserve the ability to drop shadow here?

Reply:
I see, then LGTM. Drop shadows are so Material 2